web/services/getUserTemplatesService.php

```diff
@@ -22,6 +22,7 @@
 include_once(PHPREPORT_ROOT . '/web/services/WebServicesFunctions.php');
 include_once(PHPREPORT_ROOT . '/util/ConfigurationParametersManager.php');
 include_once(PHPREPORT_ROOT . '/util/DBPostgres.php');
```

Comment:
You probably don't need the two includes above, please double-check the three services :)